src/frontend/src/binder/relation/mod.rs

```diff
@@ -40,6 +42,7 @@ pub enum Relation {
     Subquery(Box<BoundSubquery>),
     Join(Box<BoundJoin>),
     WindowTableFunction(Box<BoundWindowTableFunction>),
+    GenerateSeriesFunction(Box<BoundGenerateSeriesFunction>),
```

Comment:
Well I actually considered refactoring via a more general TableFunction (also termed as SetReturningFunction in pg). 

```
#[derive(Debug)]
pub struct BoundTableFunction {
    pub(crate) func_type: ExprType,
    pub(crate) args: Vec<ExprImpl>,
    pub(crate) row_type: Vec<DataType>,
}
```

Reply:
We can do it later PR after discussion. We need some unified mechanism to do this.